Create Staircase.py

bracar opened the pull request:
###### Closing Issue/Referencing Issue
issue reference (#1 ) 

###### Changes Made In Pull Request

* Added solution to Staircase problem
 https://www.hackerrank.com/challenges/staircase/problem

ows-ali commented:
@bracar Please add your name and info in README.md and push again to your bracar:master branch

bracar commented:
I've messed up my first fork, so I deleted it, forked again and created [new PR](https://github.com/ows-ali/Hacktoberfest/pull/145) with Hackerrank solution and updated README.md with my info

ows-ali commented:
Sure. I have merged it. Thanks @bracar for your contribution
